pom.xml

```diff
@@ -22,7 +22,7 @@
   <version>1.0-SNAPSHOT</version>
   <packaging>pom</packaging>
   <properties>
-      <teamcity-version>2017.1</teamcity-version>
+      <teamcity-version>2019.2</teamcity-version>
```

Comment:
Not mandatory but better. [TW-61490](https://youtrack.jetbrains.com/issue/TW-61490) required for this build feature.